### DISCUSSION TITLE
Break down recipe dependencies into multiple lines

### DISCUSSION THREAD

**Discussion by ifraixedes** [casey/just#1419] [Q&A] [answered]
I wonder if it's possible to break down recipe dependencies into multiple lines. 

My use case is that I have a recipe that calls another recipe several times because it receives parameters and having to put all the dependencies in the same line makes the line very long.

For example:

```Makefile
deploy: (_derypt-file "some/large/path/to/the/source" "some/large/path/to/the/destination") (_decrypt-file "some/other/large/path/to/the/source" "some/other/large/path/to/the/destination") && clean-up
  echo deploying

clean-up:
  echo "cleaning up"

_decrypt-file src dst:
  echo "decrypting '{{src}}' {{dst}}'"
```

I tried to use `\` for breaking the dependencies into several lines, but it didn't work.

Thank you in advance.

**Comment by casey** [comment, marked as answer]
Sorry you're hitting this! This is a known limitation in the parser. See issue #1266, and PR #1272 which fixes it, although it looks like it's stalled for now.
